src/main/java/de/tum/in/www1/artemis/service/ResultService.java

```diff
@@ -98,10 +98,8 @@ public Result findLatestResultWithFeedbacksForParticipation(Long participationId
      * @param result
      */
     public void setAssessor(Result result) {
-        User currentUser = userService.getUserWithGroupsAndAuthorities();
+        User currentUser = userService.getUser();
         result.setAssessor(currentUser);
-        resultRepository.save(result);
```

Comment:
Why is the result no longer saved? Is this called in a Transactional Context?

Reply:
The result is saved manually in the only calling method:
c32d06e#diff-5ffcfdcc92994d424ce48cfa2becd4f4R293